src/LibMtpSharpStandardMacOS/OpenedMtpDevice.cs

```diff
@@ -13,6 +14,7 @@ namespace LibMtpSharpStandardMacOS
 {
     public class OpenedMtpDevice : IDisposable
     {
+        private const int KindleVendorId = 0x1949;
```

Comment:
What is source of this information? Maybe we can add a comment for this?
I've found something like this https://www.the-sz.com/products/usbid/index.php?v=0x1949 but they are not listed all Kindle devices. 

Do you think it might be useful also on Windows?

Reply:
This value is also used to detect Kindle device in our mobile app https://github.com/legimi/legimi.mobile/blob/sprint91_28576/Platform/src/Mobile/Legimi.Mobile.Kindle/KindleDeviceDetector.android.cs

> Do you think it might be useful also on Windows?
I think so